.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,84 @@
+name: Bug Report
+description: Broken or unintended behavior with the library.
+labels: 'bug-unconfirmed'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Before Submitting your Issue
```

Comment:
suggest the title first letter cap: Before submitting your issue

Reply:
Updated to all letter caps as title should be all letter caps: Before Submitting Your Issue